Makefile

```diff
@@ -79,7 +79,7 @@ init-db: start-db
 	bin/tools bin/post-install-dev.sh
 
 fix-cs-php:
```

Comment:
can't you add https://github.com/squizlabs/PHP_CodeSniffer/wiki/Fixing-Errors-Automatically  here ?

Reply:
done